kafka_scripts/sample_producer.py

```diff
@@ -0,0 +1,76 @@
+import random
+import time
+from confluent_kafka import avro
+from confluent_kafka.avro import AvroProducer
+
+from constants import KafkaConstants
```

Comment:
This should be `from .constants import ...` since it's a relative import rather than absolute (there is no global python module called constants).